### PR TITLE
Milestone 9

### PR DESCRIPTION
**Pull request by tshiu**
Added while loop

### CODE REVIEW

**Comment by tshiu** [comment]
@mlezeng I think I accidentally deleted the PE Exercises from my master branch and now have conflicts to be resolved. Not quite sure how do these, would love to go over this. Otherwise, the exercises and deliverables should be present!

File: PE_Exercise_1.py [hunk 1]
```diff
@@ -14,21 +15,21 @@
...
 
-	#setting response as nil now so that it can be called within the loop-like function
-	response = nil 
+#setting response as nil now so that it can be called within the loop-like function
+response = nil 
 
-	#setting path as the endpoint to put record_type from prompt into the endpoint
-	path = /v2.0/{record_type}
+#setting path as the endpoint to put record_type from prompt into the endpoint
+path = /v2.0/{record_type}
```
Comment (mlezeng) [inline PE_Exercise_1.py]: Can you update this to match Python's format for adding variables?

File: PE_Exercise_1.py [hunk 2]
```diff
@@ -14,21 +15,21 @@
...
 
-	#will set client ID
-	client_ID = [Client_ID_for_district]
-	#will set client secret 
-	client_secret = [Client_Secret_for_district]
+#will set client ID
+client_ID = [Client_ID_for_district]
```
Comment (mlezeng) [inline PE_Exercise_1.py]: for district or app?

Also, you may need to query for this data.

File: ex33.py [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+#SD
+#1. Conver while-loop to a function you can call
+
+def print_numbers_to_6():
+	
+	i = 0 
+
+	the_list = []
+
+	while i < 6:
```
Comment (mlezeng) [inline ex33.py]: The 6 in this line should have been replaced by a variable per the study drill

Comment (mlezeng) [reply ex33.py]: This doesn't have to be fixed since I see you did this below

File: ex35.py [hunk 4]
```diff
@@ -0,0 +1,126 @@
+from sys import exit
+
+#defining the function that is entitled gold_room
+def gold_room():
+	print "This room is full of gold. How much do you take?"
+
+	#setting the variable next to the raw input that should answer the question before
+	next = str(raw_input("> "))
+	# If the string in next contains either 1 or 0, we will set the variable how_much
```
Comment (mlezeng) [inline ex35.py]: I think you described what the original function did, then updated the function to improve it. Can you update your comment to show what the new function does?

File: ex36.py [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+
+	wig_fallen = False 
+
+	while True:
+		next = raw_input("> ")
+
+		if next == "clip":
+			sashay_away("Always just throw off the wig!")
+
+		elif next == "throw" and not wig_fallen:
```
Comment (mlezeng) [inline ex36.py]: Is there a case where wig_fallen would be set to true? If so, where? If not, why include this?

File: m9del.py [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+		
+	elif guess > number: 
+		print "A little lower..."
+
+	else:
+		break
+		
+	new_guess = int(raw_input("Lets try again. What's your guess? \n"))
+
+	if new_guess == guess:
```
Comment (mlezeng) [inline m9del.py]: Hmm....if the user keeps entering the same number each time, guess should not go up. Can you rewrite this the reflect that?

Hint: maybe there should be another loop here!

File: m9del.py [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+		guess = new_guess
+		guess_taken = guess_taken + 1
+		#print "this is the elif block"
+		#print "The is the new_guess: %d " % new_guess
+		#print "This is the number: %d " % number
+		#print "This is the guess: %d " % guess
+
+
+
+else:
```
Comment (mlezeng) [inline m9del.py]: Does this run? Since this else line is tabbed back to start, it's not considered part of the loop. I think this line shouldn't be here?

**Review by mlezeng — CHANGES_REQUESTED** [review]

File: ex36.py (hunk 5 above)
Comment (tshiu) [reply ex36.py]: Oops, I think I was thinking of making things more complicated but then forgot about it. I removed the wig_fallen variable

File: m9del.py (hunk 6 above)
Comment (tshiu) [reply m9del.py]: added a while loop instead!

File: m9del.py (hunk 7 above)
Comment (tshiu) [reply m9del.py]: adjusted the tab

File: ex35.py (hunk 4 above)
Comment (tshiu) [reply ex35.py]: Updated!

File: PE_Exercise_1.py (hunk 1 above)
Comment (tshiu) [reply PE_Exercise_1.py]: Done, I believe

File: ex33.py (hunk 3 above)
Comment (tshiu) [reply ex33.py]: Oops, adjusted

File: PE_Exercise_1.py (hunk 2 above)
Comment (tshiu) [reply PE_Exercise_1.py]: App! yes, I wasn't quite sure how to query for it but will make a note to add this as we go along

**Comment by tshiu** [comment]
@mlezeng  I believe I made the changes. let me know!

**Review by mlezeng — APPROVED** [review]

**Pull request closed without merging**